### PR TITLE
New package: aide-0.16.2

### PR DESCRIPTION
**Pull request by abenson**

### CODE REVIEW

**Comment by abenson** [comment]
Hm, wait a sec, the two directories should be created.

**Comment by abenson** [comment]
Alright, that should cross-build.

File: srcpkgs/aide/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Template file for 'aide'
+pkgname=aide
+version=0.16.2
+revision=1
+build_style=gnu-configure
+configure_args="--with-mhash --with-posix-acl --with-prelink --with-xattr
+ --with-zlib --with-e2fsattrs --disable-static"
+conf_files="/etc/aide.conf"
+make_dirs="/var/lib/aide 0750 root root /var/log/aide 0750 root root"
```
Comment (daniel-eys) [inline srcpkgs/aide/template]: There should be a newline separating the two make_dirs entries.

> By convention, there is only one entry of `dir perms user group` per line.

(Snippet from the make_dirs paragraph of Manual.md)

Comment (abenson) [reply srcpkgs/aide/template]: Ah, I misread that line.

File: srcpkgs/mhash/template [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+short_desc="Uniform interface to a large number of hash algorithms"
+maintainer="Andrew Benson <abenson+void@gmail.com>"
+license="LGPL-2.1-or-later"
+homepage="http://mhash.sourceforge.net/"
+distfiles="$SOURCEFORGE_SITE/project/mhash/mhash/${version}/mhash-${version}.tar.bz2"
+checksum=56521c52a9033779154432d0ae47ad7198914785265e1f570cee21ab248dfef0
+
+mhash-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}"
```
Comment (xtraeme) [inline srcpkgs/mhash/template]: `depends="${sourcepkg}>=${version}_${revision}"`

**Comment by abenson** [comment]
Fixed.

File: srcpkgs/aide/template [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+pkgname=aide
+version=0.16.2
+revision=1
+build_style=gnu-configure
+configure_args="--with-mhash --with-posix-acl --with-prelink --with-xattr
+ --with-zlib --with-e2fsattrs --disable-static"
+conf_files="/etc/aide.conf"
+make_dirs="/var/lib/aide 0750 root root
+ /var/log/aide 0750 root root"
+hostmakedepends="flex bison"
```
Comment (xtraeme) [inline srcpkgs/aide/template]: `flex` depends on `bison`, `bison` is redundant.

Comment (abenson) [reply srcpkgs/aide/template]: Was avoiding implicit dependencies. From the README:

> o GNU flex.
> o GNU yacc (bison).

Comment (abenson) [reply srcpkgs/aide/template]: Fixed.

**Pull request merged**